### DISCUSSION TITLE
Add more predefined gradients for color scheme

### DISCUSSION THREAD

**Discussion by KBac** [grafana/grafana#38230] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Add more predefined gradients for color scheme, e.g. red-yellow-green

**Why is this needed**:
This is really nice to have predefined gradients for color scheme. But there are only few of them. E.g. there is green-yellow-red. But sometimes I need to invert it to use red-yellow-green. Of cause I can create my own using thresholds, but it's not as fancy as the gradient one, and I need to define a lot of thresholds to get a good result.
Also it would be nice to have an option to reverse the existing gradients.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
